news/fitcontribution-dep.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* Deprecate camel case method formatting in ``FitContribution``.
```

Comment:
please can you list all hte public facing methods that were depracated as a matter of policy.  This is user facing so it is more useful to them than which module from a dev point of view that you worked on.